pink_annotation_schema.ttl

```diff
@@ -111,11 +111,40 @@ foaf:logo rdf:type owl:AnnotationProperty .
 
 ###  https://w3id.org/pink#conformance
 :conformance rdf:type owl:AnnotationProperty ;
+             rdfs:range [ rdf:type owl:Class ;
+                          owl:oneOf ( :mandatory
+                                      :optional
+                                      :recommended
+                                    )
+                        ] ;
              dcterms:description "Whether the annotation is mandatory, recommended or optional as part of the PINK Annotation Schema"@en ;
              vann:usageNote "It MUST take one of the values 'mandatory', 'recommended' or 'optional'."@en ;
              rdfs:isDefinedBy <https://w3id.org/pink> ;
-             skos:prefLabel "conformance" ;
-             rdfs:range xsd:string .
+             skos:prefLabel "conformance" .
+
+
+#################################################################
+#    Classes
+#################################################################
+
+
+#################################################################
+#    Individuals
+#################################################################
+
+###  https://w3id.org/pink#mandatory
+:mandatory rdf:type owl:NamedIndividual ;
```

Comment:
Should we add a rdfs:comment or vann:usageNote to these individuals, explaining how they are supposed to be used?